### ISSUE TITLE
Write docs for configuring these converters

### ISSUE THREAD

**Issue by jilleJr** [jilleJr/Newtonsoft.Json-for-Unity.Converters#25]
## Description

Continuation of #17, but depending on #16 that wont be in the first release.

## Motivation

Need the documentation for users to start using this package.

## Suggested solution

Needs documentation in wiki:

- How to configure via the ScriptableObject described in #16

**Comment by jilleJr** [comment]
Closing as duplicate of #41
